subgraph.yaml

```
@@ -0,0 +1,37 @@
+specVersion: 0.0.4
+description: TAP Protocol allows senders to deposit escrow for specific recievers
+repository: https://github.com/semiotic-ai/timeline-aggregation-protocol-contracts
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: Escrow
+    network: mainnet
+    source:
+      address: '0xfdfad30ae5e5c9Dc4fb51aC35AB60674FcBdefB3'
+      abi: Escrow
+      startBlock: 0
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      file: ./src/mappings/Escrow.ts
```

This typo breaks the `codegen` step
```suggestion
      file: ./src/mappings/escrow.ts
```